crates/nargo_cli/tests/execution_success/brillig_to_le_bytes/src/main.nr

```diff
@@ -1,14 +1,13 @@
 use dep::std;
 
-unconstrained fn main(x : Field) -> pub [u8; 4] {
+unconstrained fn main(x : Field) -> pub [u8; 31] {
     // The result of this byte array will be little-endian
     let byte_array = x.to_le_bytes(31);
-    let mut first_four_bytes = [0; 4];
-    for i in 0..4 {
-        first_four_bytes[i] = byte_array[i];
+    assert(byte_array.len() == 31);
+
+    let mut bytes = [0; 31];
+    for i in 0..31 {
+        bytes[i] = byte_array[i];
     }
-    // Issue #617 fix
```

Comment:
This issue is legacy code. We previously always returned a bit/byte array of a set size (https://github.com/noir-lang/noir/blob/90abce80c25c4224380daad2bef85fcbe2c00e4b/noir_stdlib/src/lib.nr#L12), and had this check for us incorrectly mapping it. Now that we specify a specific limb count mapping is unnecssary